src/utilities/isSuppotedByPrettier.js

```diff
@@ -0,0 +1,45 @@
+function isSuppotedByPrettier(languageName) {
```

Comment:
export on first line, rename file and add comments

Reply:
Done!